code/datums/outfits/outfit_admin.dm

```diff
@@ -64,14 +64,16 @@
 	if(istype(R))
 		R.set_frequency(SYND_FREQ)
 	H.faction += "syndicate"
+	H.mind.offstation_role = TRUE
```

Comment:
This will probably runtime if the mob doesn't yet have a mind, such as if they're freshly created. 

I'm not sure if there's a good way to set things up on addition of a mind here.

Reply:
Would a if(H.mind) on everything work? Ancient vampire is the only one in the admin outfit with offstation roles at the moment and it is working without creating a runtime.

Reply:
It would prevent the runtime but it would mean that people who get put into these bodies afterwards wouldn't get the off station role applied.

Reply:
It will runtime, might I suggest adding a signal to help deal with this perhaps?

If you would like assistance with handling this, I'll be free Roughly in 2 or so hours

Reply:
I have taken a look at how signals work in signals.dm and I've no clue how to use them, I would appreciate the help when you have the time.

Reply:
GND suggestion added.